enhancements/cluster-logging/forwarder-input-selectors.md

```diff
@@ -195,7 +191,7 @@ The operator will validate resources upon reconciliation of a **ClusterLogForwar
 
 
 * The **ClusterLogForwarder** CR defines a valid spec
-* Input spec fields that are "globs" (i.e. Namespace, container) match RE: '`[a-zA-Z0-9\*]*`'
+* Input spec fields that are "globs" (i.e. Namespace, container) match RE: '`^[a-zA-Z0-9\*]*$`'
 * Input field 'selector' is a valid metav1.LabelSelector 
 * Input enum fields accept only the values listed
 * type "infrastructure" include specs at least one value
```

Comment:
type "infrastructure" sources specifies at least one value

... same with line:198, do you want to change the word "include" to "sources"?